### PR TITLE
fix: fees adjusted

### PR DESCRIPTION
**Pull request by lumir-mrkva**

### CODE REVIEW

File: runtime/basilisk/src/lib.rs [hunk 1]
```diff
@@ -171,9 +171,8 @@ impl WeightToFeePolynomial for WeightToFee {
 	///   - Setting it to `0` will essentially disable the weight fee.
 	///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
 	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
-		// extrinsic base weight (smallest non-zero weight) is mapped to 1/10 CENT
-		let p = CENTS; // 1_000_000_000_000
-		let q = 10 * Balance::from(ExtrinsicBaseWeight::get()); // 7_919_840_000
+		let p = 5 * CENTS; // 1_000_000_000_000
+		let q = Balance::from(ExtrinsicBaseWeight::get()); // 7_919_840_000
```
Comment (lumir-mrkva) [inline runtime/basilisk/src/lib.rs:175]: update/remove comments

File: integration-tests/src/fees.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		let info = call.get_dispatch_info();
+		let len = call.encoded_size() as u32;
+
+		println!("info: {:?}", info);
+		println!("len: {:?}", len);
+
+		let fees = TransactionPayment::compute_fee(len, &info, 0);
+
+		//This test is not correct it's approx 1bsx of from real fees
+		println!("fees: {:?}/{:?}", fees / 1_000_000_000, 22.2105);
```
Comment (Roznovjak) [inline integration-tests/src/fees.rs]: use some constant as a denominator

File: integration-tests/src/cross_chain_transfer.rs [hunk 3]
```diff
@@ -44,11 +44,11 @@ fn transfer_from_relay_chain() {
 	Basilisk::execute_with(|| {
 		assert_eq!(
 			basilisk_runtime::Tokens::free_balance(1, &AccountId::from(BOB)),
-			10028 * UNITS / 10
+			10000 * UNITS / 10
```
[the remark this answers is not in the document]
Comment (lumir-mrkva) [reply integration-tests/src/cross_chain_transfer.rs:47]: can you investigate more? why would adjusting fee broken the test, are we not sending enough tokens to over for fees?

File: integration-tests/src/fees.rs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+use crate::kusama_test_net::*;
+use basilisk_runtime::TransactionPayment;
+use frame_support::weights::GetDispatchInfo;
+use sp_runtime::codec::Encode;
+use xcm_emulator::TestExt;
+
+#[test]
+//TODO: fix this, fees calculcated in this test are apporx 2bsx off.
+fn transaction_fees() {
```
[the remark this answers is not in the document]
Comment (lumir-mrkva) [reply integration-tests/src/fees.rs]: we are planning to expand them with measuring xcm cost etc which requires this integration setup

File: integration-tests/src/cross_chain_transfer.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (Roznovjak) [reply integration-tests/src/cross_chain_transfer.rs:48]: yes, sending more fixes it. Martin is updating it RN

**Review by jak-pan — APPROVED** [review]

**Pull request merged**